### ISSUE TITLE
Better documentation needed

### ISSUE THREAD

**Issue by staticdev** [mattwcole/gelf-extensions-logging#3]
I've made a pull request #2 to begin fixing it, but more improvements are needed.

**Comment by mattwcole** [comment]
PRs welcome! What do you feel is still missing or unclear?

**Comment by staticdev** [comment]
Is it possible to customize the server, port and protocol? I've seen your example uses udp://localhost:12201, but is not telling how to change that behavior. Does nlog.config overwrites that?

Still, concerning the framework version used, did you test this package in which versions?

**Comment by mattwcole** [comment]
UDP is the only supported protocol at the moment, and uses port 12201 by default. The readme explains how to change the server and port, and you can see the full list of options [here](https://github.com/mattwcole/gelf-extensions-logging/blob/dev/src/Gelf.Extensions.Logging/GelfLoggerOptions.cs). The library has nothing to do with NLog, so any settings in `nlog.config` are not used.

So far I have tested with up to .NET Core 1.1. I would expect 2.0 to work also, but I've not had time to try it yet. The preferred way of configuring logging with .NET Core 2.0 is to use `IServiceCollection.AddLogging()` - this will require an update to this library before we can use that though.

**Comment by staticdev** [comment]
Good, I've tried in .net core 2.0 and it worked! Later I can put some example code.

**Comment by mattwcole** [comment]
I'm closing this issue as it is a little broad. If you have specific suggestions on what is missing from the docs feel free to create another issue, or submit a PR.
